src/DetailsView/components/requirement-view.tsx

```diff
@@ -30,7 +30,7 @@ import * as React from 'react';
 import styles from './requirement-view.scss';
 
 export type RequirementViewDeps = {
-    assessmentActionMessageCreator: AssessmentActionMessageCreator;
+    getAssessmentActionMessageCreator: () => AssessmentActionMessageCreator;
```

Comment:
nit: it seems like we're not using this here; we should probably remove it entirely.